src/components/data-exploration/generic-gene-table/GeneTable.jsx

```diff
@@ -56,7 +57,10 @@ const GeneTable = (props) => {
   };
 
   const handleTableChange = (newPagination, a, newSorter) => {
-    const newTableState = { ...tableState, pagination: newPagination, sorter: { ...newSorter } };
+    const sortOrder = newSorter.order === 'ascend' ? 'ASC' : 'DESC';
+    dispatch(setGeneOrdering(newSorter.field, sortOrder));
```

Comment:
`GeneTable` is being used both in `DiffExprResults` and in `GeneListTool`, doesn't this dispatch being in here mean that there is a coupling that we don't want?

Reply:
I mean shouldn't this be in `DiffExprResults`?

Reply:
oh right... I'll move it out of `GeneTable`.

Reply:
Moved out of `geneTable` into `diffExprResults`